### PR TITLE
Extend the workflow to autogenerate the vim doc from markdown

### PR DESCRIPTION
**Pull request by leiserfg**

### CODE REVIEW

**Comment by L3MON4D3** [comment]
Good idea, definitely nice to have docs in markdown, lemme take a look a what is generated.

File: doc/luasnip.txt [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
+| Field      | Type    | Description                    |
+|------------|---------|--------------------------------|
+| `trig`     | String  |                                |
+| `name`     | String  | Name, used by other plugins    |
+| `dscr`     | String  | Description, used by other     |
+|            |         |                                |
+| `wordTrig` | Boolean | If true, the snippet is only   |
+|            |         |                                |
+| `regTrig`  | Boolean | If true, `trig` is interpreted |
+|------------|---------|--------------------------------|
```
Comment (L3MON4D3) [inline doc/luasnip.txt:67]: Could you fixup that table and insert the missing lines? (Was that formatted by stylua? Am I misreading the commit history?)

**Comment by leiserfg** [comment]
The message of the automatic commit was wrong, now It's fixed, I also fixed the table.

**Comment by L3MON4D3** [comment]
Okay, sweet, thanks a lot :D

**Pull request merged**